training/orig/4.5/src/lesson_perception/src/perception_node.cpp

```diff
@@ -60,7 +60,7 @@ int main(int argc, char *argv[])
   try
   {
     listener.waitForTransform(world_frame, camera_frame,  ros::Time::now(), ros::Duration(6.0));
```

Comment:
@jrgnicho, I purposefully left the waitForTransform time as ros::Time::now(); errors occurred when I changed that to (0). I'm not sure why this combination works, but it it is the only configuration (waitForTransform with ros time 0 and lookupTransform with ros time now) that leads to no errors in the console.